OpenRA.Game/Map/Map.cs

```diff
@@ -1416,5 +1417,10 @@ public string Translate(string key, IDictionary<string, object> args = null)
 
 			return modData.Translation.GetString(key, args);
 		}
+
+		public void Dispose()
```

Comment:
There's one other non-commandy call to a `Map` constructor in `NewMapLogic.cs` that lacks a call to `Dispose`, can we add that?